src/composables/useFlawCvssScores.ts

```diff
@@ -61,7 +69,7 @@ export function useFlawCvssScores(flaw: Ref<ZodFlawType>) {
       });
     }
 
-    return result;
+    return groupWith((a, b) => a.isHighlighted === b.isHighlighted, result);
```

Comment:
This is just to reduce the number of html nodes by grouping them (there are changes in the template to accommodate the grouping)

|Before|After|
|---|---|
|![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/b30df6de-201d-46e8-b2c5-c0a22300ea23)|![image](https://github.com/RedHatProductSecurity/osim/assets/4268580/7ae36df8-ac71-470f-bfbf-f2d264d13313)|